<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CPACSTreeWidget</class>
 <widget class="QWidget" name="CPACSTreeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="searchLineEdit">
       <property name="toolTip">
        <string>The tree is filtered by the expression typed in this bar. The expression is check against the type of the element and the uid of the element. </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="expertViewCheckBox">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the view mode of the tree. In expert mode, you have every element of the cpacs tree starting form /cpacs/vehicles/. In basic mode you have a simplify cpacs tree starting from /cpacs/vehicles/aircraft/model/. &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Expert View</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <attribute name="headerCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="headerDefaultSectionSize">
      <number>122</number>
     </attribute>
     <attribute name="headerStretchLastSection">
      <bool>true</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
